### PR TITLE
simplify the convertUnit

### PR DESCRIPTION
**Pull request by LINGLUOJUN**
## What's changed?

simplify the convertUnit

## Checklist

- [ ]  I have read the [Contributing Guide](https://hertzbeat.apache.org/docs/community/code_style_and_quality_guide)
- [ ]  I have written the necessary doc or comment.
- [ ]  I have added the necessary unit tests and all cases have passed.

## Add or update API

- [ ] I have added the necessary [e2e tests](https://github.com/apache/hertzbeat/tree/master/e2e) and all cases have passed.

### CODE REVIEW

File: collector/src/main/java/org/apache/hertzbeat/collector/dispatch/unit/impl/AbstractUnitConvert.java [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+
+package org.apache.hertzbeat.collector.dispatch.unit.impl;
+
+import lombok.extern.slf4j.Slf4j;
+import org.apache.commons.lang3.StringUtils;
+import org.apache.hertzbeat.collector.dispatch.unit.UnitConvert;
+import java.math.BigDecimal;
+import java.math.RoundingMode;
+import java.util.Map;
+
```
Comment (yuluo-yx) [inline collector/src/main/java/org/apache/hertzbeat/collector/dispatch/unit/impl/AbstractUnitConvert.java:26]: remove the extra blank lines and just keep the one. 🤔

File: collector/src/main/java/org/apache/hertzbeat/collector/dispatch/unit/impl/AbstractUnitConvert.java [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+
+/**
+ * abstract UnitConvert
+ */
+@Slf4j
+public abstract class AbstractUnitConvert implements UnitConvert {
+
```
Comment (yuluo-yx) [inline collector/src/main/java/org/apache/hertzbeat/collector/dispatch/unit/impl/AbstractUnitConvert.java:33]: ditto

File: collector/src/main/java/org/apache/hertzbeat/collector/dispatch/unit/impl/DataSizeConvert.java [hunk 3]
```diff
@@ -17,57 +17,26 @@
...
 import org.springframework.stereotype.Component;
+import java.util.Arrays;
+import java.util.Map;
+import java.util.stream.Collectors;
+
 
 /**
  * the convert of data size
  */
 @Slf4j
```
Comment (yuluo-yx) [inline collector/src/main/java/org/apache/hertzbeat/collector/dispatch/unit/impl/DataSizeConvert.java:31]: Maybe we can remove or add some logs? There's no log output here.

Comment (LINGLUOJUN) [reply collector/src/main/java/org/apache/hertzbeat/collector/dispatch/unit/impl/DataSizeConvert.java:31]: got it

File: collector/src/main/java/org/apache/hertzbeat/collector/dispatch/unit/impl/AbstractUnitConvert.java (hunk 1 above)
Comment (LINGLUOJUN) [reply collector/src/main/java/org/apache/hertzbeat/collector/dispatch/unit/impl/AbstractUnitConvert.java:26]: got it,and already fixed it

File: collector/src/main/java/org/apache/hertzbeat/collector/dispatch/unit/impl/AbstractUnitConvert.java (hunk 2 above)
Comment (LINGLUOJUN) [reply collector/src/main/java/org/apache/hertzbeat/collector/dispatch/unit/impl/AbstractUnitConvert.java:32]: got it,and already fixed it

**Comment by LINGLUOJUN** [comment]
@yuluo-yx @tomsun28  can it be merged?

**Pull request merged**